MAUI/Busy-Indicator/AnimationType.md

```diff
@@ -161,3 +161,34 @@ SfBusyIndicator busyIndicator = new SfBusyIndicator()
 The following gif image illustrates the result of the above code.
 
 ![Double Circle](Images/AnimationType/DoubleCircle.png)
+
+### Global
```

Comment:
change it to Globe

Reply:
I have updated the enum AnimationType from Global to Globe.